### PR TITLE
Fix translation issues, tweak styling, and upgrade to Ant Design 5

### PR DESCRIPTION
**Pull request by SanjeevLakhwani**
- fixed translation issue on tooltip
- fixed translation issue on insufficient data message
- header buttons rounded
- Upgraded to antdv5
- optimized tabpane rendering to be more scalable

### CODE REVIEW

File: src/js/components/Overview/charts/BentoPie.js [hunk 1]
```diff
@@ -41,8 +41,9 @@ function BentoPie({ data, height }) {
 
   // Changing immutable data to mutable data
   data = [...data];
-  // remove "missing" field if zero
+
   data = data.filter((e) => !(e.x === 'missing'));
+  data = data.map((d) => ({ ...d, x: t(d.x) }));
```
Comment (davidlougheed) [inline src/js/components/Overview/charts/BentoPie.js:46]: I would avoid these reassignments; imo its not any less readable in the form of:

```js
data = [...data]
    .filter((e) => !(e.x === 'missing'))
    .map((d) => ({ ...d, x: t(d.x) }));
```
with indenting. In general avoiding mutation of the variable (reference) value is good, although this does get mutated later a bunch anyway...

File: src/js/components/Overview/charts/BentoPie.js [hunk 2]
```diff
@@ -120,7 +121,7 @@ function BentoPie({ data, height }) {
         <path d={`M${sx},${sy}L${mx},${my}L${ex},${ey}`} stroke={fill} fill="none" />
         <circle cx={ex} cy={ey} r={2} fill={fill} stroke="none" />
         <text x={ex + (cos >= 0 ? 1 : -1) * 12} y={ey + 3} textAnchor={textAnchor} style={currentTextStyle}>
-          {labelShortName(t(name))}
+          {labelShortName(name)}
```
Comment (davidlougheed) [inline src/js/components/Overview/charts/BentoPie.js:124]: for my own understanding, why is this now untranslated?

File: src/js/components/TabbedDashboard.js [hunk 3]
```diff
@@ -30,27 +29,48 @@ const TabbedDashboard = () => {
...
+    },
+    {
+      title: 'Provenance',
+      content: <ProvenanceTab />,
+      loading: false,
+      key: 'provenance',
+    },
+  ];
+
+  const MappedTabPanes = TabPanes.map(({ title, content, loading, key }) => ({
```
Comment (davidlougheed) [inline src/js/components/TabbedDashboard.js:66]: this shouldn't be name-formatted like it's a component

File: src/js/components/TabbedDashboard.js [hunk 4]
```diff
@@ -30,27 +29,48 @@ const TabbedDashboard = () => {
...
   const isFetchingSearchFields = useSelector((state) => state.query.isFetchingFields);
+  const individuals = useSelector((state) => state.data.individuals);
+
+  const individualCount = (
+    <Title level={5}>
+      {t('Individuals')}: {individuals}
+    </Title>
+  );
+
+  const TabPanes = [
```
Comment (davidlougheed) [inline src/js/components/TabbedDashboard.js:45]: this shouldn't be name-formatted like it's a component

**Review by davidlougheed — CHANGES_REQUESTED** [review]

File: src/js/components/Overview/charts/BentoPie.js (hunk 2 above)
Comment (SanjeevLakhwani) [reply src/js/components/Overview/charts/BentoPie.js:124]: It is translated earlier so no need after

Comment (davidlougheed) [reply src/js/components/Overview/charts/BentoPie.js:124]: maybe put a comment mentioning where this comes from/where it gets translated to make it easy to understand in the future then?

**Comment by SanjeevLakhwani** [comment]
Dealt with the requested changes!

**Review by davidlougheed — APPROVED** [review]

**Comment by davidlougheed** [comment]
this should, when released, be versioned 0.7.0 when combined with Brennan's PR

**Pull request merged**